### DISCUSSION TITLE
Crashing release detail screen

### DISCUSSION THREAD

**Discussion by ceskyDJ** [community/community#64297] [Mobile] [answered]
### Select Topic Area

Bug

### Body

Hi,

I found a problem with the release detail screen. It works well mostly, but for the last release of Obtainium it crashes. I think the best option will be seeing my screen recording: https://www.loom.com/share/a287b28840c048b58f6408b3aec79c03.

Thanks for reading my bug report and possibly fixing the issue. Have a nice day!

HW/SW details:

- Device: Samsung Galaxy S22
- OS: Android 13
- App version: 1.126.0

**Comment by M-YasirGhaffar** [comment]


**Reply by ceskyDJ** [reply]
I've just tried it again, and it seems to work well. So, the only thing I have for you is the linked video.

**Comment by Jaber-Saed** [comment]
I appreciate your bug report and the provided screen recording. Unfortunately, I don't have direct access to view external content like the video you've shared. However, I can provide you with general steps and suggestions on how to address issues like the one you've described:

1. **Reproduce the Issue:**
   - As a first step, ensure that you can consistently reproduce the issue on your device. This will help you isolate the specific conditions under which the crash occurs.

2. **Check for Updates:**
   - Make sure that both your app and device's software are up to date. Sometimes, app crashes can be resolved by updating to the latest version.

3. **Clear App Cache and Data:**
   - Go to your device's settings, find the app (in this case, "Obtainium"), and clear its cache and data. Then, restart the app and see if the issue persists.

4. **Check for Known Issues:**
   - Visit the app's official website, support forums, or any relevant social media accounts to see if there are any known issues or solutions for the specific problem you're facing.

5. **Contact App Support:**
   - If the issue persists, reach out to the app's support team. They should have a way to report bugs or provide assistance. In your case, since you're using "Obtainium," check if there's a support email, website contact form, or in-app help option to submit a bug report.

6. **Provide Detailed Information:**
   - When reporting the issue, provide as much detail as possible. Describe the steps you took that led to the crash, any error messages you received, and if there are any patterns or specific scenarios that trigger the problem.

7. **App Permissions:**
   - Ensure that the app has the necessary permissions to function properly on your device. Sometimes, permission issues can lead to crashes.

8. **Device Restart:**
   - Restart your device. Sometimes, a simple restart can resolve temporary glitches.

9. **Uninstall and Reinstall:**
   - If none of the above steps work, you might need to uninstall the app and then reinstall it. Be sure to back up any important data associated with the app before doing this.

10. **Update Android WebView:**
   - Sometimes, issues related to rendering content can be caused by the WebView component on Android. Ensure that your Android WebView is up to date via the Google Play Store.

If you're still unable to resolve the issue after trying these steps, your best option is to contact the app's support team directly. They will have access to more specific information about the app's behavior and can offer assistance tailored to your situation.

**Reply by ceskyDJ** [reply]
It's interesting you can't view a publicly shared video (tested from incognito mode of Chrome). I shared a lot of similar videos using Loom to many people and no one has this problem. So, it will be good to announce this to Loom team, so they can fix it. I hope you'll do it (if you haven't yet).

I don't have a time for this today. I think I will have it in a few days.

**Comment by ceskyDJ** [comment, marked as answer]
I can't reproduce it anymore, so I close this discussion. I don't know, why it didn't work and now it does.
